README.md

```diff
@@ -71,6 +73,8 @@ an issue or PR, they are ignored otherwise.
 The `GITHUB_TOKEN` secret is required to interact and authenticate with the GitHub API to open
 the pull request. The example is [deployed here](https://github.com/vsoch/pull-request-action-example) with an example opened (and merged) [pull request here](https://github.com/vsoch/pull-request-action-example/pull/1) if needed.
```

Comment:
@mszhanyi I'm wondering if there is a way to make this less complicated. E.g., the user still could use the GITHUB_TOKEN variable defined in secrets to be a PAT? Are there cases when both would be wanted so this wouldn't work?

And the same question for `GITHUB_REPOSITORY`. Is there a way to not have two variables for the "same" thing?

Reply:
According to https://docs.github.com/en/actions/reference/environment-variables#naming-conventions-for-environment-variables , It looks it's not necessary to define GITHUB_XXX.